backend/generateData/generateData.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require('mongoose')
```

Comment:
call it generateDummyData, I would like to move this folder into utils or something as well